### PR TITLE
Make wheel slices clickable

### PR DESCRIPTION
**Pull request by AlexPD93**
When you click on a slice it will display a component related to the specific title and give resources related to that title.

### CODE REVIEW

File: components/ui/PieChart.tsx
Comment (that-Gui) [inline components/ui/PieChart.tsx:1]: please move this component to its own folder

File: app/page.tsx
Comment (that-Gui) [inline app/page.tsx:1]: remove this from the landing page and onto its own component

File: components/CustomSheet.tsx
Comment (that-Gui) [inline components/CustomSheet.tsx:1]: this is not how you use this component

**Review by that-Gui — CHANGES_REQUESTED** [review]
changes are required before we can merge this

**Review by Feargh — APPROVED** [review]
Excellent work.

**Pull request merged**